src/scripts/browserDeviceProfile.js

```diff
@@ -932,12 +932,16 @@ export default function (options) {
 
     if (supportsHdr10(options)) {
         hevcVideoRangeTypes += '|HDR10';
+        // Should also be able to play DoVi with HDR10 fallback
+        hevcVideoRangeTypes += '|DOVIWithHDR10';
```

Comment:
As [mentioned before](https://github.com/jellyfin/jellyfin/pull/10469/files#diff-ab579a24be3882376b30a1f50886a7fdd3e80b5c4b18ec33e20f83ee8caafe66R1809-R1810), you cannot add them globally, which will cause Server to enable `dvh1` fourcc for HLS.js in Chrome and thus playback will fail.

Reply:
The code here means that DOVI Profile 7/8 support is also reported for devices that only support HDR10/HLG. This is not true. The report will be wrong. You should split the three checks in [canPlayDolbyVisionHevc](https://github.com/jellyfin/jellyfin-web/blob/master/src/scripts/browserDeviceProfile.js#L213-L219) and report `DOVIWithHDR10/HLG` accordingly.

Reply:
web report (HDR10) => server tag DV 8.1 as `hvc1` and drop the dovi metadata => web play as HDR10